packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -552,8 +552,8 @@ const RedirectToInstantMeetingModal = ({ expiryTime }: { expiryTime?: Date }) =>
               */}
 
               <p className="font-medium">{t("please_do_not_close_this_tab")}</p>
-              <div className="h-[450px]">
-                <iframe className="mx-auto h-full" src="https://cal.games/" />
+              <div className="mt-4 h-[414px]">
```

Comment:
I already added margin top here on my PR https://github.com/calcom/cal.com/pull/13258/

Reply:
can you push these changes on my PR feat: instant meeting improvements #13258?